src/AppInstallerCLITests/TestData/Manifest-Good-SystemReferenceComplex.yaml

```diff
@@ -4,14 +4,10 @@ Version: 1.7.32
 Publisher: Microsoft
 License: MIT License
 Description: A complex system reference test file.
-InstallerType: Zip
+InstallerType: EXE
 PackageFamilyName: Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
 ProductCode: "{Foo}"
 Installers:
-  - InstallerType: Zip
-    Arch: x86
-    Url: https://rubengustorage.blob.core.windows.net/publiccontainer/msixsdkx86.zip
-    Sha256: 69D84CA8899800A5575CE31798293CD4FEBAB1D734A07C2E51E56A28E0DF8C82
```

Comment:
I'm assuming you made sure that we don't lose coverage by removing this?

Reply:
Yes, the v1.4 manifest tests cover this scenario and ensure that fields from the root get passed to the installer if they are not defined.